usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/internal/reporting/JBossToolsComponents.java

```diff
@@ -76,6 +76,7 @@
 		MODESHAPE("org.jboss.tools.modeshape.rest.feature", "org.jboss.tools.modeshape.rest"),
 		OPENSHIFT("org.jboss.tools.openshift.express.feature", "org.jboss.tools.openshift.express.client"),
 		PORTLET("org.jboss.tools.portlet.feature", "org.jboss.tools.portlet.core"),
+		QUARKUS("org.jboss.tools.quarkus.feature", "org.jboss.tools.quarkus.runtime"),
```

Comment:
I would chose org.jboss.tools.quarkus.core for the bundle as runtime is about embedding Quarkus which is using a pre release of Quarkus and may change drastically (chance of removal) in the future